locales/en.default.schema.json

```
@@ -1214,6 +1214,15 @@
         "name": "Image with text"
       }
     },
+    "main-account": {
+      "name": "Account"
+    },
+    "main-activate-account": {
```

```suggestion
    "account-activate": {
```
Nitpick but doesn't have to be changed. Just a thought.